### PR TITLE
Add issue template for new features

### LINKED ISSUE #4: Create a template for non-bug issues
We lack an appropriate template to describe roadmap issues/feature requests. At the moment, we only have one for bugs.
It should include the problem we want to solve, a proposed solution, potential challenges, and unknowns. To be better defined.

One potential downside is that our users may use the feature request template, which conflicts with the current policy of collecting feature requests through discussions.
To me, this is not a big deal; on the contrary, I would prefer having feature requests and bugs in the same place.

### PR DESCRIPTION
**Pull request by chrean**
## Summary

Fixes #4 

Add a tentative template for new non-bug issues.

### CODE REVIEW

**Review by kennyadsl — COMMENTED** [review]
Nothing to say about the content. Maybe we can follow the other issue templates format (eg. https://raw.githubusercontent.com/solidusio/.github/a1f6a2b8517bdc3932cde966583e5716bc7ff9c3/.github/ISSUE_TEMPLATE/bug_report.md)?

File: .github/ISSUE_TEMPLATE/new_feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Desired Behavior
+
+Describe desired behavior for improvements/enhancements.
+What is the problem we want to solve?
+What is the plan?
```
Comment (elia) [inline .github/ISSUE_TEMPLATE/new_feature:7]: nit: wrapping the explanatory text in `<!--` HTML comments `-->` will save anyone opening a new issue the time for deleting said text making it visible only in editing mode

File: .github/ISSUE_TEMPLATE/new_feature [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Actual Behavior
```
Comment (elia) [inline .github/ISSUE_TEMPLATE/new_feature]: `Actual` is surely correct, but maybe, since we're not addressing bugs, we should use `Current` instead?

Comment (elia) [inline .github/ISSUE_TEMPLATE/new_feature]: Although there's always a way to fill it somehow, I feel it's not always relevant for new features, so I would also suggest this section to be optional, not a blocker tho.

**Review by elia — COMMENTED** [review]
Thanks for adding this, left a couple of comments/nits 👍

File: .github/ISSUE_TEMPLATE/new_feature (hunk 1 above)
Comment (chrean) [reply .github/ISSUE_TEMPLATE/new_feature:7]: Good point, done.

File: .github/ISSUE_TEMPLATE/new_feature (hunk 2 above)
Comment (chrean) [reply .github/ISSUE_TEMPLATE/new_feature]: Makes sense, done.

Comment (chrean) [reply .github/ISSUE_TEMPLATE/new_feature]: Good point, done. ff02fce

**Review by elia — APPROVED** [review]
Simplicity FTW 🎯

**Comment by chrean** [comment]
@kennyadsl I think the format for bugs would be an overkill for this kind of issue, primarily aimed at us writing our roadmap issues. Not sure we need all those details, although they are valuable information for bugs. What do you think?

**Comment by kennyadsl** [comment]
@chrean sure, I didn't mean the content but the format of headings, using HTML comments (as Elia suggested as well) and similar things.

**Comment by chrean** [comment]
@kennyadsl that makes sense, I have just changed the format of headings to be the same as bug requests.

**Comment by kennyadsl** [comment]
@chrean nice, can you squash the commits into a single one, please?

**Pull request merged**